lua/entities/gmod_wire_pod.lua

```diff
@@ -456,11 +456,13 @@ function ENT:Think()
 			if (self.Relative) then
 				local originalangle
 				if (self.RC) then
-					originalangle = ply.InitialAngle
+					originalangle = self.RC.InitialAngle
```

Comment:
Probably need to remove whatever was setting ply.InitialAngle before. Also it seems possible that self.RC.InitialAngle could be nil if the remote control was never turned on. Might need to check for that case.

Reply:
That line of code should only ever be reachable after SWEP:On is called, so that shouldn't be an issue, and InitialAngle isn't referenced anywhere else in the code.

Reply:
So it was always nil before? Guess that was the issue. Good find.